### PR TITLE
Ew mobile

### PR DESCRIPTION
**Pull request by tracylihuizhang**

### CODE REVIEW

File: src/pages/EWFormPage/index.js [hunk 1]
```diff
@@ -94,6 +101,49 @@ const referrals = [
...
+}
+
+const inputMobileStyle = {
+  width: "74vw",
+  color: "#8CC9BA",
+  borderRadius: 10,
+  border: "2px solid #8CC9BA",
+  backgroundColor: "#FFF",
+  height: "14vh",
+  fontSize: "12px",
```
Comment (jaismith) [inline src/pages/EWFormPage/index.js:119]: too small!

File: src/pages/EWFormPage/index.js [hunk 2]
```diff
@@ -94,6 +101,49 @@ const referrals = [
...
+}
+
+const inputMobileStyle1 = {
+  width: "74vw",
+  color: "#8CC9BA",
+  borderRadius: 10,
+  border: "2px solid #8CC9BA",
+  backgroundColor: "#FFF",
+  height: "19vh",
+  fontSize: "11.5px",
```
Comment (jaismith) [inline src/pages/EWFormPage/index.js:130]: again, too small

File: src/pages/EWFormPage/index.js (hunk 1 above)
Comment (jaismith) [inline src/pages/EWFormPage/index.js:119]: too small

File: src/pages/EWFormPage/index.js [hunk 3]
```diff
@@ -94,6 +101,49 @@ const referrals = [
...
+}
+
+const inputMobileStyle2 = {
+  width: "74vw",
+  color: "#8CC9BA",
+  borderRadius: 10,
+  border: "2px solid #8CC9BA",
+  backgroundColor: "#FFF",
+  height: "10vh",
+  fontSize: "12px",
```
Comment (jaismith) [inline src/pages/EWFormPage/index.js:142]: ^^

**Review by jaismith — CHANGES_REQUESTED** [review]
This looks awesome, thank you for all the great work!

One small change I think we should make before merging in: the smallest font size in mobile web design should usually be 16pt, we're going down to 11.5px currently in some of the descriptive text. Would you bump that up to 16pt? I've commented in a few places I see small font sizes, but I think some of this is just the default font-size from desktop trickling down.

File: src/pages/EWFormPage/index.js (hunk 2 above)
Comment (tracylihuizhang) [reply src/pages/EWFormPage/index.js:130]: I agree 16px size is great for desktop, but on mobile, due to the limited screen space, I wanted to compact things so as to fit all the components in a reasonable space proportionality 

This is what it looks like when the font size is larger (placeholders cut off at the bottom and textarea would need to be made a lot larger if the font was bigger), let me know your thoughts: 
Screen Shot 2020-09-04 at 10 52 58 PM
Screen Shot 2020-09-04 at 10 53 27 PM

**Review by kristiechow — APPROVED** [review]
Looks good to me after seeing Tracy's screenshot of what bigger font would look like

**Pull request merged**